Lab2.1/LevenshteinTest.java

```diff
@@ -0,0 +1,10 @@
+import org.junit.jupiter.api.Assertions;
+
+class LevenshteinTest {
+
+    @org.junit.jupiter.api.Test
+    void levQWERTY() {
+        Assertions.assertEquals(1.5, Levenshtein.LevQWERTY("kot", "kita"));

```

Comment:
@repcus Im bardziej atomowy test tym lepiej, więc można byłoby rozbić to na 2 metody